Tests/DependencyInjection/Compiler/ExtensionCompilerPassTest.php

```diff
@@ -250,7 +250,7 @@ public function testProcess()
         $extensions = $def->getExtensions();
         $this->assertCount(4, $extensions);
 
-        $this->assertInstanceOf(get_class($this->securityExtension), $extensions[0]);
```

Comment:
You are not missing something.
The previous assert checks that there are 4 extensions.
The extension with index 0 was the security extension, however after applying the sorting it is now on index 1. Index 0 contains AdminEventExtension which is not a part of this test. It is a global extension that is added.

I see no BC breaks in this however. If you think otherwise please let me know.

Thanks